relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics.snap

```diff
@@ -3349,7 +3370,6 @@ expression: metrics
             "environment": "fake_environment",
             "release": "1.2.3",
             "span.description": "Cold Start",
-            "span.group": "2d675185edfeb30c",
```

Comment:
This looks like you need to also extract the group for app starts in addition to the known groups. Or make app start spans a known group.